crates/cli-support/src/js/mod.rs

```diff
@@ -800,7 +800,7 @@ impl<'a> Context<'a> {
                                 return await WebAssembly.instantiateStreaming(module, imports);
 
                             }} catch (e) {{
-                                if (module.headers.get('Content-Type') != 'application/wasm') {{
+                                if (!module.headers || module.headers.get('Content-Type') !== 'application/wasm') {{
```

Comment:
```suggestion
                                if (typeof module.headers === 'undefined' || module.headers.get('Content-Type') !== 'application/wasm') {{
```
I would prefer to be more explicit here.
Otherwise LGTM!